NwPluginAPI/Loader/AssemblyLoader.cs

```diff
@@ -226,5 +239,56 @@ private static bool TryGetAssembly(string path, out Assembly assembly)
 			assembly = null;
 			return false;
 		}
+		
+		/// <summary>
+		/// Attempts to load Embedded assemblies (compressed) from the target
+		/// </summary>
+		/// <param name="target">Assembly to check for embedded assemblies</param>
+		private static void ResolveAssemblyEmbeddedResources(Assembly target)
+		{
+			Log.Debug($"Attempting to load embedded resources for {target.FullName}", Log.DebugMode);
+
+			
+			var resourceNames = target.GetManifestResourceNames();
+			foreach (var name in resourceNames)
+			{
+				Log.Debug($"Found {name}", Log.DebugMode);
+				if (name.EndsWith(".dll"))
+				{
+					using (MemoryStream stream = new MemoryStream())
+					{
+						Log.Debug($"Loading {name}", Log.DebugMode);
+						var dataStream = target.GetManifestResourceStream(name);
+						if (dataStream == null)
+						{
+							Log.Error($"Unable to resolve {name} Stream was null");
+							continue;
+						}
+
+						dataStream.CopyTo(stream);
+						Assembly.Load(stream.ToArray());
+						Log.Debug($"Loaded {name}", Log.DebugMode);
+					}
+				}
+				else if (name.EndsWith(".dll.compressed"))
```

Comment:
```suggestion
				else if (name.EndsWith(".dll.compressed", StringComparison.OrdinalIgnoreCase))
```

Maybe let's use `StringComparison.OrdinalIgnoreCase` to avoid potential issues?